Filesystem.cpp

```diff
@@ -25,23 +25,30 @@ bool FileSystem::hasFile(string arg){
     }
     return false;
 }
-
-
-void FileSystem::touch(vector <string> args){
-    bool hasD = hasDir(args.front());
-    bool hasF = hasFile(args.front());
+bool FileSystem::checkDocsExits(string docName){
+    bool hasD = hasDir(docName);
+    bool hasF = hasFile(docName);
     if(hasD || hasF){
         if(hasD){
             cout<<"There is already a folder with this name!"<<endl;
-            return;
+            return true;
         }
         if(hasF){
             cout<<"The file is already exits!"<<endl;
-            return;
+            return true;
         }
     }
-    currentDir->makeFile(args.front());
-    return;
+    return false;
+}
+
+void FileSystem::touch(vector <string> args){
+    bool docExits = checkDocsExits(args.front());
+    if(!docExits){
+        currentDir->makeFile(args.front());
+    }
+    else {
+        return;
+    }
```

Comment:
Ennek itt igy nem sok ertelme maradt. Viszont az nem biztos,hogy jo, hogy a checkolo fuggveny irogat ki. Gondolj bele, ha kesobb valamihez TE csak sajat magnak le akarod ellenorizni, hogy letezik-e egy konyvtar, vagy file, akkor random megjelennek a felhasznalonak ilyen uzenetek. En ezt athoznam ide a touch-ba, bar teny, hogy akkor nemcsak ide, hanem 10 masik fuggvenybe is be kell irni. Ezt meg lehet oldani szepen egy dobott Exceptionnel, amiben benne van a hibauzenet, es akkor a kulso fuggveny eldonti majd, hogy kiirja-e vagy sem. De ha ezt nem akarod, akkro legalabb legyen egy verbose argumentuma a chek-nek, ami defaulton false, vagy ilyesmi.